BuilderDesignPattern/src/main/java/com/designPattern2/Builder.java

```diff
@@ -0,0 +1,13 @@
+package com.designPattern2;
+
+/**
+ * Builder interface defines all possible ways to configure a product.
+ */
+
+public interface Builder {
+    
+    void setBuildingType(BuildingType type);

```

Comment:
What would happen if you were to reference this method outside of the `com.designPattern2` package?

Reply:
I thought you could just import the Package and the class by doing something like:

import com.designPattern2.Builder